server/routes/clubProfileRoutes.js

```diff
@@ -0,0 +1,13 @@
+const fns = require('../controllers/clubProfileController')
```

Comment:
change the name of fns to something like functions so this code is a bit more clear/easier to read